src/recipe_dictionary.h

```diff
@@ -105,6 +105,9 @@ class recipe_subset
             const std::string &cat,
             const std::string &subcat = std::string() ) const;
 
+        /** Get all recipes */
+        std::vector<const recipe *> get_all() const;
```

Comment:
Still, it's not needed. `recipe_dictionary` already has a `begin`/`end` pair of iterators.